pentagon/component/kops/__init__.py

```diff
@@ -0,0 +1,84 @@
+from pentagon.component import ComponentBase
+import os
+import glob
+import shutil
+import logging
+import traceback
+import sys
+import re
+import subprocess
+
+from pentagon.helpers import render_template
```

Comment:
We don't have a strict style-guide but a common default Python one is PEP8 which has the [import listed](https://www.python.org/dev/peps/pep-0008/#imports) as:

```
standard library imports
related third party imports
local application/library specific imports
```

In this case the first import is for `ComponentBase`  which is local and (according to PEP8) would go at the bottom. 

Standard layouts for imports help to keep a consistent codebase.